src/components/Listings/Listings.js

```diff
@@ -1,6 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './Listings.scss';
 
-export default Listings = () => {
-  return ()
-}
\ No newline at end of file
+const Listings = (props) => {
+  const filteredListings = props.listings.filter(listing => listing.area_id === props.selectedAreaId)
```

Comment:
Is it worth filtering these listing before they reach the `Listings` component? Then all this component has to worry about is plopping them on the page.

This isn't a blocker, but just a different way to think about things. Should the `Listings` component know about _all_ of the listings or just the ones that it cares about?

Reply:
We were having some problems trying to filter it pre-loading the listings. I don't remember the exact problem, but that is definitely the preferred logic I think.